src/main/java/de/tum/in/www1/artemis/web/rest/ModelingAssessmentResource.java

```diff
@@ -191,13 +220,23 @@ public ModelingAssessmentResource(JsonAssessmentRepository jsonAssessmentReposit
 
         Result result = updateManualResult(resultId, exerciseId, modelingAssessment, true);
         Long submissionId = result.getSubmission().getId();
+        // add assessment to compass to include it in the automatic grading process
         compassService.addAssessment(exerciseId, submissionId, modelingAssessment);
 
         return ResponseEntity.ok(result);
     }
 
     //TODO: move more logic into a new ModelingAssessmentService
 
+    /**
+     * Updates result and set attributes according to rated state.
```

Comment:
The comment is not quite detailed. The method is called "updateManualResult" so the comment should also reflect what 'manual' actually means here